### DISCUSSION TITLE
What info does Google share with TEAMMATES?

### DISCUSSION THREAD

**Discussion by damithc** [TEAMMATES/teammates#11185] [Q&A] [answered]
Received this query from a user: 

> I shared this tool with our department of management. They are concerned with the account activation process for faculty.They said that they were asked to share their Google information including access to their Google docs. I don't recall being asked to do this but It has been awhile since I set up my account. Can you clarify for us what sharing requirements you require for faculty setting up accounts using teammates.

Has there been any change in this requirement recently? Is there a way to for us to prove that or for them to verify themselves?

**Comment by wkurniawan07** [comment, marked as answer]
> They said that they were asked to share their Google information including access to their Google docs.

Do they have proof to back this up?

Under GAE SDK (i.e. current production server) authentication is managed via their proprietary Users API, so no one knows for sure. But I don't believe they require anything more than read-only access to most basic user information.

In V8 this setup is explicit within the code base.

**Reply by damithc** [reply]
![image](https://user-images.githubusercontent.com/1673303/121291704-4bec2800-c91b-11eb-8d87-e04a35eeb00b.png)

Managed to produce the above by creating a new TEAMMATES account for myself.

Posting it here for future reference, in case another user has the same question.
